lte/gateway/c/oai/lib/s8_proxy/s8_client_api.cpp

```diff
@@ -183,10 +215,8 @@ void send_s8_create_session_request(
   OAILOG_FUNC_IN(LOG_SGW_S8);
   magma::feg::CreateSessionRequestPgw csr_req;
 
-  OAILOG_INFO_UE(
-      LOG_SGW_S8, imsi64,
-      "Sending create session request for context_tied " TEID_FMT "\n",
-      sgw_s11_teid);
+  std::cout << "Sending create session request for for IMSI: " << imsi64
```

Comment:
Let's try using OAILOG to consolidate all logs in MME log file. For some of the existing implementation, it is split between MME log and syslog, which is not ideal.